### PR TITLE
Yarvis frontend ncarrasco

### PR DESCRIPTION
**Pull request by ncglobant**
Gradient texture change according to new design.

### CODE REVIEW

File: js/common/Background.js [hunk 1]
```diff
@@ -18,8 +18,12 @@ var DisplayWallBackground = (function () {
     var nParticleDriftSpeed = opts.YARVIS_HEIGHT / 4000;
 
     var particleSprite = THREE.ImageUtils.loadTexture('/assets/images/background/Bokeh.png');
-    var overlayGradient = THREE.ImageUtils.loadTexture('/assets/images/background/grad1c.png');
-
+    
+	var gradientTextureHeight = 1000;
```
Comment (felipe-sanchez) [inline js/common/Background.js]: fix indentation to use convention that is used in file, use a single var when possible separated by commas.
you already have a height defined that you can use opts.YARVIS_HEIGHT which matches the needed texture height.
Width should be 1 or 2.

File: js/common/Background.js [hunk 2]
```diff
@@ -80,7 +84,8 @@ var DisplayWallBackground = (function () {
       self.planeCollection.add(particle);
     };
 
-    self.bgScene.add(self.planeCollection);
+	// Commented the particle effects insertion while it is decided what to do.
```
Comment (felipe-sanchez) [inline js/common/Background.js]: fix indentation, though you could remove the comment. Particles where removed and can be put back in place if needed in the future by cherry picking old commits.

File: js/common/Background.js [hunk 3]
```diff
@@ -203,6 +208,28 @@ var DisplayWallBackground = (function () {
 
     requestAnimationFrame(render)
   };
+  
+  	DisplayWallBackground.prototype.generateVerticalLinearGradientTexture = function(aTextureWidth ,aTextureHeight, aTopColor, aBottomColor) {
```
Comment (felipe-sanchez) [inline js/common/Background.js]: fix indentation.
add always a space between an anonymous function and parameters.
remove space after aTextureWidth
add a space before aTextureHeight.
if line gets too long (ideally beyond 80 characters insert a new line with indentation.
just use textureWidht, textureHeight, topColor, bottomColor as parameter names, without the a before them

**Comment by andrescalandriglobant** [comment]
I think this file is outdated, you should pull and merge master and resolve all the conflicts if any

File: js/common/Background.js [hunk 4]
```diff
@@ -203,6 +208,28 @@ var DisplayWallBackground = (function () {
...
+	
+		canvas = document.createElement( 'canvas' );
```
Comment (felipe-sanchez) [inline js/common/Background.js]: group all variable definitions with a single call to var. remove commented gradient.addColorStop(0,"black/white") lines

**Comment by felipe-sanchez** [comment]
We will hold this pull request until after the call. For future reference, lets try to keep coding conventions and formatting.

**Comment by felipe-sanchez** [comment]
Pull from master and resolve merge conflicts

**Pull request closed without merging**